math/number_of_positive_divisors.cpp

```diff
@@ -2,7 +2,7 @@
  * @file
  * @brief C++ Program to calculate number of divisors
  *
- * This algorithm use the prime factorization approach.
+ * This algorithm uses the prime factorization approach.
  * Any number can be written in multiplication of its prime factors.
  * <br/>Let N = P1^E1 * P2^E2 ... Pk^Ek
```

Comment:
This can be better formatted using LaTeX syntax as such:
```suggestion
 * <br/>Let \f$N = p_1^{e_1} \times p_2^{e_2} \times\cdots\times p_k^{e_k}
```